test/literals/literal-numeric-expression.js

```diff
@@ -14,32 +14,74 @@
  * limitations under the License.
  */
 
-var assertEsprimaEquiv = require('../assertions').assertEsprimaEquiv;
+var expect = require("expect.js");
+var parse = require("../..").default;
+var assertEsprimaEquiv = require("../assertions").assertEsprimaEquiv;
+var assertParseFailure = require("../assertions").assertParseFailure;
+var expr = require("../helpers").expr;
 
 describe("Parser", function () {
   describe("literal numeric expression", function () {
+
     // Numeric Literals
     assertEsprimaEquiv("0");
     assertEsprimaEquiv("3");
     assertEsprimaEquiv("5");
     assertEsprimaEquiv("42");
+    assertEsprimaEquiv("\n    42\n\n");
+
     assertEsprimaEquiv(".14");
     assertEsprimaEquiv("3.14159");
+
     assertEsprimaEquiv("6.02214179e+23");
     assertEsprimaEquiv("1.492417830e-10");
-    assertEsprimaEquiv("0x0");
-    assertEsprimaEquiv("0x0;");
     assertEsprimaEquiv("0e+100 ");
     assertEsprimaEquiv("0e+100");
+
+    assertEsprimaEquiv("0x0");
+    assertEsprimaEquiv("0x0;");
     assertEsprimaEquiv("0xabc");
     assertEsprimaEquiv("0xdef");
     assertEsprimaEquiv("0X1A");
     assertEsprimaEquiv("0x10");
     assertEsprimaEquiv("0x100");
     assertEsprimaEquiv("0X04");
+
     assertEsprimaEquiv("02");
     assertEsprimaEquiv("012");
     assertEsprimaEquiv("0012");
-    assertEsprimaEquiv("\n    42\n\n");
+
+    // Binary Numeric Literal
+    expect(expr(parse("0b0"))).to.eql({ type: "LiteralNumericExpression", value: 0 });
```

Comment:
Let's use the `LiteralNumericExpression` AST node constructor in these positions.